.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -4,6 +4,7 @@ set -e
 
 LOGFILE="test-log"
```

Comment:
I think LOGFILE was used for pr-commenter. If it's no longer being used, we can safely remove the variable and coded that use 'LOGFILE'. @subhashkhileri - can you confirm?

Reply:
@josephca This is required. It's log file where html log report is stored.
like : [Link](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/janus-idp_backstage-showcase/1170/pull-ci-janus-idp-backstage-showcase-main-e2e-tests/1781348181875888128/artifacts/e2e-tests/janus-idp-backstage-showcase/artifacts/)